bitrise.yml

```diff
@@ -836,10 +836,10 @@ workflows:
 
             ./firefox-ios/l10n-screenshots.sh en-US
         title: Generate screenshots
-    - deploy-to-bitrise-io@2:
+    - save-cache@1:
         inputs:
-        - deploy_path: l10n-screenshots-dd/
-        - is_compress: 'true'
+        - key: l10n-screenshots-dd-$GIT_CLONE_COMMIT_HASH
```

Comment:
Would the commit hash appropriate to be the key of the cache?

Reply:
I don't know if this will work... the builds are triggered atuomatically, they will not have a commit hash if I'm not mistaken...

Reply:
Both L10nBuild and L10nScreenshot calls `git-clone` at the beginning. `GIT_CLONE_COMMIT_HASH` is exposed by `git-clone`.
https://devcenter.bitrise.io/en/references/available-environment-variables.html

Reply:
okay, if that's the case, let's try.. once you merge this please run the taskcluster trigger to start the whole process and make sure it works